style_editor.html

```diff
@@ -274,7 +482,7 @@
 </style>
 
 
-<script id="vertex-shader" type="x-shader/x-vertex">
+<script id="vertex_shader" type="x-shader/x-vertex">
```

Comment:
Universalizing: ID's use underscores, classes use dashes

Reply:
universalizing should be a separate PR